modules/cid_checker/main.tf

```diff
@@ -0,0 +1,72 @@
+data "aws_caller_identity" "current" {}
+
+#### AWS s3 bucket for market deals. ####
+
+resource "aws_s3_bucket" "cid_checker_market_deals" {
+  bucket        = "marketdeals-${var.bucket_name}"
```

Comment:
Is it possible to follow our common name convention for the resource name as we have for other resources?
the instance: "${module.generator.prefix}-marketdeals-${var.bucket_name}"
where after applying the code we should have a bucket name like `filecoin-dev-apn1-glif-marketdeals-cidchecker`.

The main goal of it increase resource association understanding and make a unique name with the same part of the bucket and for different environments.

Reply:
Was changed bucket name to "${module.generator.prefix}-marketdeals-${var.bucket_name}" by main file.